frontend/online-shop/src/App.tsx

```diff
@@ -1,26 +1,30 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import { TableCards } from "./components/productsList/TableCards";
+import { ProductDetails } from "./components/productDetails/ProductDetails";
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Redirect,
+} from "react-router-dom";
+import { ThemeProvider } from "@material-ui/styles";
+import { standardTheme } from "./themes/StandardTheme";
+import { CssBaseline } from "@material-ui/core";
 
-function App() {
+export const App = () => {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <ThemeProvider theme={standardTheme}>
+      <CssBaseline>
+        <Router>
+          <Switch>
```

Comment:
What if the user is directly accessing a path that you don't have in your switch? For example `http://localhost:4200/asdfg`. For such cases you need to have a wildcard route that catches all the paths that do not fit with any other declared route and return a 404 page.